README.md

```diff
@@ -10,7 +10,11 @@ This project has two main components:
 
 - Install rustup via brew
```

Comment:
Maybe we should just say "follow the instructions on https://rustup.rs/ to install rustup" so that we don't end up out of date?

Reply:
I like that, but I don't like piping curl into a shell... if the shell script says `rm -rf /foo/bar` but the connection is broken after `rm -rf /`, somebody will have an awful day!

Reply:
mmm - that homepage does link to other installation options to suit contributors preferences?  I just grepped their https://sh.rustup.rs script and it seems it doesn't actually call `rm` anywhere...

Reply:
Fair enough, updated :-)